flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaChangelogTableITCase.java

```diff
@@ -131,7 +133,8 @@ public void testKafkaDebeziumChangelogSource() throws Exception {
 		tEnv.executeSql(sourceDDL);
 		tEnv.executeSql(sinkDDL);
 		TableResult tableResult = tEnv.executeSql(
-			"INSERT INTO sink SELECT name, SUM(weight) FROM debezium_source GROUP BY name");
+			"INSERT INTO sink SELECT FIRST_VALUE(origin), name, SUM(weight) "
+				+ "FROM debezium_source GROUP BY name");
```

Comment:
Could you make this case more complex? For example, declaring 2 format metadata and 2 connector metadata, but only select 1 format metadata and 1 connector metadata. This helps to verify format and connector metadata can work together and the metadata pruning also works.

Reply:
Good suggestion. I found a bug.